ui/src/app/apps/apps-register/apps-register.component.html

```diff
@@ -39,7 +39,7 @@ <h1 [ngBusy]="busy">Register Applications</h1>
                 <td [class.has-error]="type.invalid && type.touched">
                   <select #type="ngModel" name="type_{{index}}" [(ngModel)]="item.type" required>
                     <option [ngValue]="item.type" disabled="disabled">Please select a type...</option>
-                    <option *ngFor="let applicationType of applicationTypes" [ngValue]="applicationType">{{applicationType | capitalize}}</option>
+                    <option *ngFor="let applicationTypeKey of applicationType.getKeys()" [ngValue]="applicationTypeKey">{{applicationType[applicationTypeKey] | capitalize}}</option>
```

Comment:
Figured this one out.  the filter in the getKeys function of ApplicationType excludes the digit 0.  My workaround was as follows:
```
   /**
   * Returns an array containing the numric key values of all
   * ApplicationType enums.
   */
  export function getKeys(): number[] {
    return Object.keys(ApplicationType).filter(isNumber).map(value => Number(value));
  }

  function isNumber(element, index, array) {
    return !isNaN(element);
  }
```

If you agree I can update on merge.